hw2/02-search.js

```diff
@@ -0,0 +1,65 @@
+// Add your code here
+document.getElementById("searchButton").addEventListener("click", handleClick);
+
+// Card class
+function createCard(character, search) {
+  const cardCol = document.createElement("div");
+  cardCol.className = "col-4 bg-light mb-4";
+  const card = document.createElement("div");
+  card.className = "card p-4";
+
+  // Create each element individually
+  const cardBody = document.createElement("div");
+  cardBody.className = "card-body";
+
+  // Handle highlighting
+  const highlight = character.name.replace(
+    new RegExp(search, "gi"),
+    (match) => `<mark class="bg-warning">${match}</mark>`
```

Comment:
Interesting use of the warning color! I did not include a color in my innerHTML while working on Ex3 which is likely why it didn't work. Now I know that color would have to be specified

Reply:
This was as close to yellow from bootstrap classes, but I would recommend using a style and setting it to yellow: ```background-color: yellow;```